coin_prices_from_coingecko/README.md

```diff
@@ -19,7 +19,7 @@ In this example, you'll learn how to:
 ## Setup
 
 - Install the Blocky AS CLI by following the
-  [setup instructions](https://blocky-docs.redocly.app/attestation-service/v0.1.0-beta.6/setup)
+  [setup instructions](https://blocky-docs.redocly.app/attestation-service/{{AS_VERSION}}/setup)
```

Comment:
I just noticed that the link to the error handling example above is missing a slash. I was going to issue a PR, but going through the repo, I only see the problem in this file, so might be quicker for you to just fix it here.

Reply:
fixed.